assets/queries/terraform/aws_bom/rds/query.rego

```diff
@@ -0,0 +1,95 @@
+package Cx
+
+import data.generic.common as common_lib
+import data.generic.terraform as tf_lib
+
+CxPolicy[result] {
+	resource := input.document[i].resource.aws_db_instance[name]
+
+	accessibility := get_accessibility(resource)
+
+	bom_output = {
+		"resource_type": "aws_db_instance",
+		"resource_name": tf_lib.get_specific_resource_name(resource, "aws_db_instance", name),
+		"resource_engine": resource.engine,
```

Comment:
`resource.engine` is required unless a` snapshot_identifier` or `replicate_source_db` is provided